### ISSUE TITLE
Lots of tracks missing

### ISSUE THREAD

**Issue by Rainbowsixteen** [grimsi/accservermanager#22]
Hi!
I'm missing lots of tracks and would be very happy if they could be added :)
Examples are Spa, Silverstone, Brands Hatch, Barcelona, Zandvoort, all the 2019-tracks
 and possibly some more.

Best regards
//Emil

**Comment by grimsi** [comment]
Duplicate of #21

Also, please note that I'm currently not maintaining this project, but feel free to write some code and open a pull request :)

**Comment by Rainbowsixteen** [comment]
Oh, that's very sad cause the application package has huge potential!
Unfortunately the only language I'm comfortable with is php and I can deal with some MySQL and MSSQL, but that's about it for databases :( Best of luck with your thesis and I hope you will return and update this awesome project in the future!

**Comment by grimsi** [comment]
Thanks a lot :)
Sadly I don't think I will continue development, at least not in the near future.
I don't play ACC any more that's what motivated me in the first place.
Also I currently have other personal projects I want to finish first.
I did my best to write clean and maintainable code and I also documented almost every important aspect in the wiki, so it should be pretty easy to get into developing this application (given the potential developer is already familiar with Angular and Java Spring)
